I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 8944, in script
    jump final_chiciaAA2
ScriptError: could not find label 'final_chiciaAA2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 8944, in script
    jump final_chiciaAA2
  File "C:\Users\sonic\Downloads\renpy-8.2.1-sdk.7z\renpy-8.2.1-sdk\renpy\ast.py", line 1698, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\sonic\Downloads\renpy-8.2.1-sdk.7z\renpy-8.2.1-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'final_chiciaAA2'.

Windows-10-10.0.19041 AMD64
Ren'Py 8.2.1.24030407
School link 1.0
Sat Nov 16 12:54:13 2024
